stripe/src/test/java/com/stripe/android/model/CardTest.java

```diff
@@ -18,6 +19,7 @@
  * Test class for {@link Card}.
  */
 @RunWith(RobolectricTestRunner.class)
+@Config(sdk = 23)
```

Comment:
same here as above -- target sdk is now 25, so let's aim the tests at that. (Won't make a difference in these very basic tests, but good form anyhow)